jme3-lwjgl3/src/main/java/com/jme3/input/lwjgl/GlfwKeyInput.java

```diff
@@ -87,6 +87,16 @@ public void invoke(long window, int key, int scancode, int action, int mods) {
 
                 keyInputEvents.add(event);
             }
+
+            @Override
+            public void close() {
```

Comment:
Is there a need to override these methods if the superclass implementation is used?